sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Administration library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-administration</artifactId>
-    <version>4.0.4</version>
+    <version>4.1.0-beta.3</version>
```

Comment:
README version should point to stable.

Reply:
I think the ones that will be included with a beta release's tag should point to the beta version, after releasing said version I'll update the READMEs to point to stable.

Reply:
sounds good